bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/texteditor/FindReplaceOverlayFirstTimePopup.java

```diff
@@ -96,7 +96,6 @@ public static void displayPopupIfNotAlreadyShown(Shell shellToUse) {
 							true)
 					.delay(POPUP_VANISH_TIME.toMillis()).open();
 		}
-
```

Comment:
Please revert this unnecessary formatting change.

Reply:
Yes! https://github.com/eclipse-platform/eclipse.platform.ui/commit/f4a575c53678d57f08428e30573dc666f79cebbd